examples/rsp-cra-18/src/JournalList.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import AddCircle from '@spectrum-icons/workflow/AddCircle';
+import {Flex, Text, Button, Form, TextArea, Picker, Item, Divider} from '@adobe/react-spectrum'
+import JournalEntries from './JournalEntries'
+
+function JournalList(){
+    let [rating, setRating] = React.useState<React.Key>('');
+    let [entryList, setEntryList] = React.useState<{rate: React.Key, description: string, id: number}[]>([]);
+    let [value, setValue] = React.useState('');
+    let [count, setCount] = React.useState(0);
```

Comment:
technically we could add multiple items to the list at once in which case they'd all end up with the same "count/id" value
better to use a `useRef` for the count so that it's always up to date